endpoints/openrtb2/auction.go

```diff
@@ -2320,8 +2320,8 @@ func (deps *endpointDeps) processStoredRequests(requestJson []byte, impInfo []Im
 // parseImpInfo parses the request JSON and returns impression and unmarshalled imp.ext.prebid
 func parseImpInfo(requestJson []byte) (impData []ImpExtPrebidData, errs []error) {
 	if impArray, dataType, _, err := jsonparser.Get(requestJson, "imp"); err == nil && dataType == jsonparser.Array {
-		_, err = jsonparser.ArrayEach(impArray, func(imp []byte, _ jsonparser.ValueType, _ int, err error) {
-			impExtData, _, _, err := jsonparser.Get(imp, "ext", "prebid")
+		_, _ = jsonparser.ArrayEach(impArray, func(imp []byte, _ jsonparser.ValueType, _ int, _ error) {
+			impExtData, _, _, _ := jsonparser.Get(imp, "ext", "prebid")
```

Comment:
Instead of ignoring possible errors in `jsonparser.ArrayEach` and `jsonparser.Get`, can we add to the array and return instead?

Reply:
I'm trying to do it (as below) - but then multiple tests like `TestJsonSampleRequests/Asserts_we_return_400s_on_requests_that_are_not_supposed_to_pass_validation#25` start to fail.  Maybe we could do it in a follow-up PR?
```go
// parseImpInfo parses the request JSON and returns impression and unmarshalled imp.ext.prebid
func parseImpInfo(requestJson []byte) (impData []ImpExtPrebidData, errs []error) {
	impArray, dataType, _, err := jsonparser.Get(requestJson, "imp")
	if err != nil {
		errs = append(errs, err)
		return
	}
	if dataType == jsonparser.Array {
		_, err = jsonparser.ArrayEach(impArray, func(imp []byte, _ jsonparser.ValueType, _ int, _ error) {
			if err != nil {
				// errs = append(errs, err)
			}
			impExtData, _, _, _ := jsonparser.Get(imp, "ext", "prebid")
			var impExtPrebid openrtb_ext.ExtImpPrebid
			if impExtData != nil {
				if err := jsonutil.Unmarshal(impExtData, &impExtPrebid); err != nil {
					errs = append(errs, err)
				}
			}
			newImpData := ImpExtPrebidData{imp, impExtPrebid}
			impData = append(impData, newImpData)
		})
	}
	return
}
```

Reply:
> Maybe we could do it in a follow-up PR?

Sure @dmitris. Thank you for looking into this